src/tasks/AndroidAppBuilder/Templates/monodroid.c

```diff
@@ -225,13 +225,17 @@ mono_droid_runtime_init (const char* executable, int managed_argc, char* managed
 
     // TODO: set TRUSTED_PLATFORM_ASSEMBLIES, APP_PATHS and NATIVE_DLL_SEARCH_DIRECTORIES
 
-    const char* appctx_keys[2];
+    const char* appctx_keys[3];
     appctx_keys[0] = "RUNTIME_IDENTIFIER";
     appctx_keys[1] = "APP_CONTEXT_BASE_DIRECTORY";
+    appctx_keys[2] = "System.TimeZoneInfo.LocalDateTimeOffset";
```

Comment:
Are we considering this as "internal-use only"? Or would we expect developers to get this AppContext setting?

With its current name, it kind of seems like an "official" setting. I wonder if we should rename it to better indicate that we don't expect others to use it. (and quite possibly we could remove it in the future if we refactor this code.)

Reply:
I believe we want to set the MonoVMProperty on Xamarin-Android at all times. And to address https://github.com/dotnet/runtime/pull/74459#discussion_r956348984, I believe we would want to maintain alignment of AndroidAppBuilder in `dotnet/runtime` with what is done in Xamarin-Android over any blanket cost that might be added to all apps. However, I did some simple timing using `System.currentTimeMillis()` and `Instant.now()` and it seems instantaneous.

I think the only purpose for this property is to get the fast path local date time offset, but there was a concern whether we needed this to follow runtime config naming https://github.com/dotnet/runtime/pull/74459#discussion_r953947814.